packages/manager/modules/manager-pci-common/src/hooks/useHas3AZ/useHas3AZ.ts

```diff
@@ -1,10 +1,19 @@
 import { useContext } from 'react';
-import { PCICommonContext } from '../../contexts/PCICommonContext/PCICommonContext';
+import {
+  hasMetaProperty,
+  InternalMeta,
+  PCICommonContext,
+  PCICommonMetaType,
+} from '@/contexts/PCICommonContext/PCICommonContext';
+
+function hasMetaHas3AZ(
```

Comment:
nitpick: the double "has" in the function name is weird, i would suggest renaming it to something more straightforward such as "has3AZ"

Reply:
I can rename it to `hasMetaPropertyHas3AZ` if it is better but renaming it has3AZ wouldn't be the same function